doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -878,6 +939,11 @@ Thread
 
 The issues in this section are related to the :ref:`ug_thread` protocol.
 
+.. rst-class:: v2-8-0
+
+KRKNWK-19628: On nRF54H20, persistence of keys is not yet supported
+  After rebooting the device, all persistent keys (Thread's network key) will be lost.
+
```

Comment:
Could you add it to Matter chapter as well?

Reply:
We do not usually have the same issue in several places.

Reply:
> Maybe we could say that this issue also applies to Matter.

Sounds good